iavl_tree.go

```diff
@@ -133,28 +133,6 @@ func (t *IAVLTree) HashWithCount() ([]byte, int) {
 	return t.root.hashWithCount()
 }
 
-// Save writes the tree to disk, if it was created with a datastore.
```

Comment:
:)

IAVLTree now just works in memory by itself, right?
Must use VersionedTree for persistence

Reply:
Yeah exactly. I still don't think it's perfect because IAVLTree still has an ndb in its struct, but taking that out is somewhat complicating because of the ability for the tree to lazy-load.